test/e2e/e2e_test.go

```diff
@@ -19,9 +20,10 @@ import (
 )
 
 const (
-	serviceCAOperatorNamespace   = operatorclient.OperatorNamespace
-	serviceCAOperatorPodPrefix   = operatorclient.OperatorNamespace // Same as operator namespace
-	serviceCAControllerNamespace = operatorclient.TargetNamespace
+	serviceCAOperatorNamespace = operatorclient.OperatorNamespace
+	serviceCAOperatorPodPrefix = operatorclient.OperatorNamespace // Same as operator namespace
+	//serviceCAControllerNamespace = operatorclient.TargetNamespace
```

Comment:
Don't forget to remove this change :slightly_smiling_face:

Reply:
yup, thanks, done